deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -231,7 +233,13 @@ def merge_resource_options_provider(
                         pk_size_bytes += pk_size
 
     # total data downloaded + primary key hash column + primary key column + dict size for merge
```

Comment:
Can we update the comment here accordingly?

Reply:
done.